recipes/libpcsclite/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "libpcsclite" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LudovicRousseau/PCSC/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 856a1daed9a5ae1ac265c75ce9f81743e66a972e5be5a1d3a2196ed23f75ea26
+
+build:
+  skip: True  # [win]
+  script:
+    - ./bootstrap
+    - ./configure  # [linux]
+    - ./configure --disable-libsystemd  # [osx]
+    - make
+    # Test is here
+    - make distcheck  # [linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ cdt('systemd-devel') }}  # [linux]
+    - autoconf-archive
+    - automake
+    - flex
+    - libtool
+    - libudev  # [linux]
+    - libudev-devel-cos6-x86_64  # [not linux]
+    - libusb
+    - make
+    - pkg-config 
+    - sysroot_linux-64 2.17  # [linux64]
+  host:
+  run:
```

```suggestion
requirements:
  build:
    - {{ compiler('c') }}
    - {{ cdt('systemd-devel') }}  # [linux]
    - autoconf-archive
    - automake
    - flex
    - libtool
    - make
    - pkg-config 
    - sysroot_linux-64 2.17  # [linux64]
  host:
    - libusb
    - libudev  # [linux]
    - libudev-devel-cos6-x86_64  # [not linux]
```

Probably this package depends on libusb and libudev as libraries (not build tools) so they should go in host.

---

let me try this.